wazimap_ng/datasets/management/commands/port_admin_history.py

```diff
@@ -0,0 +1,70 @@
+import sys
+
+from django.core.management.base import BaseCommand, CommandError
+from django.core.exceptions import FieldError
+from django.contrib.admin.models import LogEntry
+from django.utils import timezone
+from django.db import transaction
+
+
+class Command(BaseCommand):
+    help = 'Port django admin log to simple history'
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+
+        log_entry_objs = LogEntry.objects.all()
+        print(f"Started process for {log_entry_objs.count()} log entry objects")
+        for log_entry in log_entry_objs:
+
+            # Get instance model
+            ct = log_entry.content_type
+            instance_model = ct.model_class()
+
+            # In case we have moved apps from model
+            # we moved around bunch of models on intial stage
+            if not instance_model:
+                continue
+            try:
+                # Get instance
+                instance = instance_model.objects.get(id=log_entry.object_id)
+            except (instance_model.DoesNotExist, FieldError):
```

Comment:
Is instance_model.DoesNotExist needed if we are checking it above?

Reply:
@michaelglenister 
yup... if there is history for deleted objects. I am certain that the Django admin log also holds info about deleted data so it is just to make sure that the script does not fail.
We also changed apps for some models. so it's just to make sure we have the object present.

FieldError is where id does not exist. One case I was able to find was Token Model
Currently, we have simple history applied only on our main models and all of them have an id